### ISSUE TITLE
Privacy Badger breaking Buffer extension

### ISSUE THREAD

**Issue by anirvan** [EFForg/privacybadgerchrome#546]
I'm using Privacy Badger 2015.8.5.1 and the [Buffer extension](https://chrome.google.com/webstore/detail/buffer/noojglkidnpfjbincgijbaiedldjfbhh) v2.12.7. When I click the Buffer extension button, it executes JavaScript bundled in the extension codebase which attempts to load an iframe containing a URL like https://buffer.com/add/?url=https%3A%2F%2Fwww.eff.org%2Fprivacybadger&text=Privacy%20Badger&version=2.12.7&placement=toolbar — which is getting blocked by Privacy Badger.

When I click on the Privacy Badger icon, it shows buffer.com as a blocked tracker, so I manually slide it to the green zone to unblock it. But when I reload the page and repeat the steps, nothing changes — it's still showing up as red/blocked, and my manual override doesn't appear to stick. (However, the manual override *does* show up in the user filter settings. So buffer.com is correctly customized to green in my user filter settings, but red when I click on Privacy Badger icon in my toolbar.)

I've reproduced this on multiple Chrome instances, have rebooted the browser, and have tried enabling and disabling Privacy Badger to see if that helps any.

To reproduce this on your system:

1. add the [Buffer extension](https://chrome.google.com/webstore/detail/buffer/noojglkidnpfjbincgijbaiedldjfbhh)
2. go to any web page
3. click on the Buffer toolbar icon
4. you should see a sign-in modal pop up — but with Privacy Badger installed, the modal never appears, and I can't figure out how to override the block

**Comment by cooperq** [comment]
Duplicate of #337
